prboom2/src/m_cheat.c

```diff
@@ -999,6 +999,24 @@ dboolean M_CheatResponder(event_t *ev)
   return false;
 }
 
+dboolean M_CheatEntered(const char* element, const char* value)
+{
+  cheatseq_t* cheat_i;
+
+  for (cheat_i = cheat; cheat_i->cheat; cheat_i++)
+  {
+    if (!strcmp(cheat_i->cheat, element) && M_CheatAllowed(cheat_i->when))
+    {    
+      if (cheat_i->arg > 0)
```

Comment:
This should be `>=` right?

Reply:
You are right. Didn't see that the arguments like pw_invulnerability could be 0.